src/client/components/app/TransactionSettings_old/index.tsx

```diff
@@ -2,6 +2,7 @@ import styled from 'styled-components';
 
 import { Card, Text, SimpleDropdown } from '@components/common';
 import { toBN, formatUsd } from '@utils';
+import React from 'react';
```

Comment:
just move it to the 1 line for import hierarchy if you can! @pentcle

Reply:
also tsx files has react by default, I think this import is wrong (ts/prettify problem?)